### ISSUE TITLE
Nested Regions

### ISSUE THREAD

**Issue by robertdfrench** [Korovasoft/Regional#1]
Suppose you want to show off a whole function _and_ separately emphasize a line within that function. Such as:

```ruby
# @region coolMethod
def myCoolMethod
  # @region PrintLikeABoss
  puts "Send some faxes"
  # @endregion
end
# @endregion
```

What would the `coolMethod` snippet look like? Would it contain the Regional directives for the `PrintLikeABoss`?

**Comment by calebwherry** [comment]
I don't think so. I would think that all nested directives would be stripped out. However, I think this gets a tad but more messy if you have options like "--strip-comments" or something, then you have to do one more pass to make sure you aren't removing a comment.
